README.md

```diff
@@ -1,6 +1,20 @@
 Absolute
-=======
+=============
+
+### A web-based order platform like siren-order for Starbucks.
```

Comment:
WDYT about we need a short but powerful phrase explaining us? for now, I will remove duplicated one. :-)